indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -290,6 +300,52 @@ class BookRoomModal extends React.Component {
         );
     }
 
+    getEventOption = (event) => {
+        const start = moment(event.startDt).format('L LT');
+        const sameDate = moment(event.startDt).isSame(event.endDt, 'date');
+        const endTime = moment(event.endDt).format('LT');
```

Comment:
The same as above.

Reply:
shouldn't it be `serializeTime` here? I know it does the same as `serializeDate` when I pass a format parameter, but the naming could make it confusing to read.

Reply:
Yeah, works fine as well. Even better ;)